GetandTake/Pages/Products/Edit.cshtml.cs

```diff
@@ -0,0 +1,61 @@
+using GetandTake.Models;
+using GetandTake.Models.DTOs.BaseDTO;
+using GetandTake.Models.DTOs.ListDTO;
+using GetandTake.Services.Abstract;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace GetandTake.Pages.Products;
+
+public class EditModel : PageModel
+{
+
+    private readonly IProductService _productService;
+
+    private readonly ICategoryService _categoryService;
+
+    private readonly ISupplierService _supplierService;
+
+    public ProductsDTO ProductDTO { get; set; }
+
+    [BindProperty]
+    public ProductDTO Product { get; set; }
+
+
+    public IEnumerable<Category> Categories { get; private set; }
+
+    public IEnumerable<Supplier> Suppliers { get; private set; }
+
+    public EditModel
```

Comment:
Usually there is no need to add separate wrappings for constructor brackets. The following format is quite appropriate:
```suggestion
    public EditModel (
        IProductService productService,
        ICategoryService categoryService,
        ISupplierService supplierService)
        {
             ....
         }
```